src/api/json/catalog.json

```diff
@@ -1727,6 +1727,14 @@
       ],
       "url": "https://raw.githubusercontent.com/ory/hydra/master/.schema/version.schema.json"
     },
+    {
+      "name": "ide.host.json",
+      "description": "JSON schema for Visual Studio template host file",
```

Comment:
SInce this is 'ide.host.json', it's not specific to "Visual Studio" maybe we can replace "Visual Studio" with "IDE"?